REQUIRE

```diff
@@ -6,3 +6,4 @@ StatsBase
 Distributions
 DataFrames
 KernelDensity
+Loess
```

Comment:
Sorry I missed this comment.  Yes adding dependencies should not be done lightly, but this is probably acceptable.